packages/sdk-rtl/src/baseTransport.ts

```diff
@@ -173,7 +173,7 @@ export abstract class BaseTransport implements ITransport {
     let signaller;
     if (AbortSignal.timeout) {
       const ms = sdkTimeout(options) * 1000;
-      let signaller = AbortSignal.timeout(ms);
+      signaller = AbortSignal.timeout(ms);
```

Comment:
Removed double let declaration, since it's block scoped, and would cause a new blocked scoped variable, which shadowed the top level `signaller` but the blocked scoped value was never included in props, since it's destroyed as soon as this block is executed.